MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Getting started with .NET MAUI ProgressBar control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI ProgressBar (Progress Bar) control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Getting started with .NET MAUI ProgressBar (Progress Bar)
+
+This section explains the steps required to add the progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill and track fill. This section covers only basic features needed to know to get started with Syncfusion progress bar.
```

Comment:
This section explains the steps required to add the progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion progress bar.

Reply:
This section explains the steps required to add the progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill.

this changes alone was included

Reply:
done